scripts/connect

```diff
@@ -3,6 +3,5 @@
 display=":$(ls /tmp/.X11-unix/* | sed 's#/tmp/.X11-unix/X##' | head -n 1)"
 user=$(who | grep '('$display')' | awk '{print $1}' | head -n 1)
 uid=$(id -u $user)
-sleep 0.1
```

Comment:
why did you remove this?

Reply:
It seems to work without it, and sleeping is sort of an anti-pattern. If something needs to be awaited, then a synchronous call or checking a variable/value should be used.

Reply:
It didn't work without it for me, that's why I added it. Netctl calls those scripts before it has actually updated the profile data so we need to sleep a bit so we can say which profile was started